corehq/apps/sms/models.py

```diff
@@ -2851,3 +2851,22 @@ def create_for_domain_and_date(cls, domain, date):
             cls.objects.create(domain=domain, date=date)
         except IntegrityError:
             pass
+
+
+class Email(models.Model):
+    """
+    Represents an email that is associated with a messaging subevent.
+    """
+
+    domain = models.CharField(max_length=126, db_index=True)
+    date = models.DateTimeField(null=True, db_index=True)
```

Comment:
Arggh, I missed this `null=True` because I wasn't looking at the first argument. Do you think it's worth changing, and if so, can I do a quick PR to modify it since this hasn't been run or the proper way of creating another migration file?

Reply:
I'd revert the other PR, then re-revert it and add a commit with the change